.circleci/generate_config.py

```diff
@@ -351,7 +368,42 @@ def add_test_definition_jobs_to_workflow(
             jobs.append(create_test_job(test, False, build_config, build_job))
 
 
-def add_test_jobs_to_workflow(workflow, tests, build_config, build_job, repl2):
+def add_rta_ui_test_jobs_to_workflow(args, workflow, build_config, build_job):
+    jobs = workflow["jobs"]
+    ui_testsuites = [
+        "UserPageTestSuite",
+        "CollectionsTestSuite",
+        "ViewsTestSuite",
+        "GraphTestSuite",
+        "QueryTestSuite",
+        "AnalyzersTestSuite",
+        "DatabaseTestSuite",
+        "LogInTestSuite",
+        "DashboardTestSuite",
+        "SupportTestSuite",
+        "ServiceTestSuite",
+    ]
+    if args.ui_testsuites != "":
+        ui_testsuites = args.ui_testsuites.split(',')
+    deployments = ['SG'
+                   #"CL",
+                   ]
```

Comment:
Why is "CL" commented out?

Reply:
should we run both by default or not? since its doubling the number of jobs. Basically a management decision.